Documentation/sources/push-notification/automatic-handling-and-tracking.md

```diff
@@ -0,0 +1,85 @@
+                    MobileCore.setPushIdentifier(token);
+                }       
+            }
+        });
+    }
+}
+```
+
+## Display and tracking push notification
+
+#### Register Messaging extension's FirebaseMessagingService
+
+> **_NOTE:_** 
+ If your application already registered a FirebaseMessagingService class, then you can skip this step and move to [Using your own FirebaseMessagingService](#using-your-own-firebasemessagingservice) section.
+
+Messaging extension uses `MessagingService` to receive, display and track push notifications. To register the `MessagingService` add the following code to the `AndroidManifest.xml` file:
+
+```java
+        <service
+            android:name="com.adobe.marketing.mobile.messaging.MessagingService"
+            android:exported="false">
+            <intent-filter>
+                <action android:name="com.google.firebase.MESSAGING_EVENT" />
+            </intent-filter>
+        </service>
+```
+
+
+#### Using your own FirebaseMessagingService
+
+If your application already registered a FirebaseMessagingService class, then you can use the following API to display and track push notifications originated from Adobe Journey Optimizer.
+
+```java
+public class YourAppFirebaseMessagingService extends FirebaseMessagingService {
+    @Override
+    public void onMessageReceived(@NonNull RemoteMessage remoteMessage) {
+        if (MessagingService.handleRemoteMessage(this, message)) {
+            // Push notification originated from Adobe Journey Optimizer is handled by the Messaging extension.
+        } else {
+            // Handle push notification originated from other sources.
+        }       
+    }
+}
+```
+
+## Notification Channel
+
+Notification channel was introduced in Android Oreo (API level 26). It allows you to group notifications and allows users to customize the notification preferences for each channel. 
+
+You must create at least one notification channel to display push notifications on devices running Android Oreo or higher. Follow the [android documentation](https://developer.android.com/develop/ui/views/notifications/channels) to create and manage notification channels. Once a notification channel is created, you can use the channel id while [designing the push notification](https://experienceleague.adobe.com/docs/journey-optimizer/using/push/design-push.html) in Adobe Journey Optimizer.
+
+**Default Channel**: If you do not create a notification channel, or if the notification channel specified while designing the push notification does not match with any of the created channels, the Messaging extension will present the push notification through its predefined fallback notification channel, which is named "General Notifications."
+
+## Configuring Small Icon
+
+Use MobileCore's API to set the small icon resource id. This icon will be used as the small icon for all the incoming push notification from Adobe Journey Optimizer.
```

Comment:
Update to: This icon will be used as the small icon for all the incoming push **notifications** from Adobe Journey Optimizer.